Code/IO/include/sitkImportImageFilter.h

```diff
@@ -118,11 +118,7 @@ class SITKIO_EXPORT ImportImageFilter : public ImageReaderBase
   // components per pixel needs to be set, otherwise the method
   // does not exist. This is done with the enable if idiom.
   template <class TImageType>
-  typename std::enable_if<!IsVector<TImageType>::Value>::type
-  SetNumberOfComponentsOnImage(TImageType *)
-  {}
-  template <class TImageType>
-  typename std::enable_if<IsVector<TImageType>::Value>::type
+  void
   SetNumberOfComponentsOnImage(TImageType *);
```

Comment:
Looks like this member function declaration (`SetNumberOfComponentsOnImage`) may be removed as well!